src/main/webapp/create-recipe-blank.html

```diff
@@ -0,0 +1,208 @@
+<!--The MIT License (MIT)
```

Comment:
Could you explain the rationale behind the name of this html? Thanks.

Reply:
It's a blank recipe creation form; there's also a create-recipe-filled for a filled-out example for the user to look at as a reference.